### DISCUSSION TITLE
Camera pan now requires Ctrl Key

### DISCUSSION THREAD

**Discussion by seijihariki** [zero01101/openOutpaint#103] [General]
Just a warning for everyone before updating, now (as of 3207e865665ed31dbfc4028f6131a5d149fd8c79) camera panning requires the use of the Control Key, like for zooming.

This is mainly to avoid confusion with mouse interactions with the tools/elements in the canvases, and for future-proofing/consistency.

**Comment by seijihariki** [comment]
@zero01101 just a quick look at something I'll probably send in today when I get home!

![Screenshot_2022-12-16-19-44-12-453-edit_com.teamviewer.teamviewer.market.mobile.jpg](https://user-images.githubusercontent.com/7552751/208201421-9dae9bb0-11d9-48a6-8fbc-d953b3e27949.jpg)

**Reply by zero01101** [reply]
👀 👀 👀 👀 👀 👀 👀 👀 👀 

~~i think this is EXACTLY what i """tried""" to do forever ago and it crushed me lmao, good luck on your quest valiant warrior o7~~

~~lmao wait never mind i think i 100% missed the context of what this is :D~~

I WAS WRONG 
[by which i mean i was right the first time](https://www.youtube.com/watch?v=wX1x7pfH8fw)
